src/package.cr

```diff
@@ -71,6 +71,28 @@ module Shards
       raise ex
     end
 
+    def install_executables
+      if installed? && spec.executables.any?
+        Dir.mkdir_p(Shards.bin_path)
+
+        spec.executables.each do |name|
+          Shards.logger.debug "Install bin/#{name}"
+          source = File.join(resolver.install_path, "bin", name)
+          destination = File.join(Shards.bin_path, name)
+
+          {% if flag?(:windows) %}
+            FileUtils.cp(source, destination)
+          {% else %}
+            if File.exists?(destination)
+              next if File.stat(destination).ino == File.stat(source).ino
```

Comment:
I'd just take this out, and either:
- always replace
- always ignore if it exists.

Reply:
We can't ignore: updating a shard may build a new executable, if shards made the hardlink, we're all set (no need to copy), but if the destination was changed for some reason, then we should delete/link again.

Reply:
Well inodes are going to be removed from `File::Info` because they're platform-specific so there's little point in having this check just to remove it later.

Let's just always hardlink.

Reply:
Sigh.

Reply:
Let's just make File::Info have mtime, file? and directory?. Let's drop everything else. Not cross platform, useless or whatever else. I'll use `stat(2)` directly to get anything done 😭

Reply:
Well:
```crystal
File.link("foo", "bar") # => 0
File.link("foo", "bar") # => Error creating link from foo to bar: File exists (Errno)
```

The only solutions are:
- delete/copy all the time (stupid, waste);
- check inodes, delete + link if they're different (smart, efficient).

I'm sorry, but I won't remove the inode check. You'll have to add `File::Info#inode` for systems that support it, so we have a compile time error on windows, for example.

Reply:
No, the correct solution is to hide the inode and supply a `File::Info#same?(File::Info)`. On unix it will check if device and inode are equal (but those fiends are still hidden). On windows it does whatever.